.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,170 @@
+            openpbta_testing: 1
+            openpbta_subset: 0
+
+          - name: Molecular Subtyping - CHORDOMA
+            entrypoint: molecular-subtyping-chordoma/run-molecular-subtyping-chordoma.sh
+            openpbta_subset: 0
+
+          - name: Molecular Subtyping - EWS
+            entrypoint: molecular-subtyping-EWS/run_subtyping.sh
+
+          - name: Molecular Subtyping - NEUROCYTOMA
+            entrypoint: molecular-subtyping-neurocytoma/run_subtyping.sh
+
+          - name: Molecular Subtyping - HGG
+            entrypoint: molecular-subtyping-HGG/run-molecular-subtyping-HGG.sh
+            openpbta_subset: 0
+
+          - name: Molecular Subtyping - LGG
+            entrypoint: molecular-subtyping-LGAT/run_subtyping.sh
+            openpbta_subset: 0
+
+          - name: Molecular Subtyping - PATHOLOGY/COMPILE
+            entrypoint: molecular-subtyping-pathology/run-subtyping-aggregation.sh
+            openpbta_testing: 1
+
+          - name: Molecular Subtyping - INTEGRATE (+ add cancer groups)
+            entrypoint: molecular-subtyping-integrate/run-subtyping-integrate.sh
+
+          # Analysis modules
+          - name: Independent Specimens
+            entrypoint: independent-samples/run-independent-samples.sh
+
+          - name: Independent Specimens pre-release
+            entrypoint: independent-samples/run-independent-samples.sh
+            run_for_subtyping: 1
+
+          - name: TP53/NF1 scores
+            entrypoint: tp53_nf1_score/run_classifier.sh
+            openpedcan_polya_strand: 0
+
+          - name: Fusion filtering
+            entrypoint: fusion_filtering/run_fusion_merged.sh
+
+          - name: Fusion summary
+            entrypoint: fusion-summary/run-new-analysis.sh
+            openpbta_subset: 0
+
+          - name: Consensus CN
+            entrypoint: focal-cn-file-preparation/run-prepare-cn-OpenTarget.sh
+            openpbta_testing: 1
+
+          - name: TMB calculation
+            entrypoint: tmb-calculation/run_tmb_calculation.sh
+
+          - name: Immune Deconvolution
+            entrypoint: immune-deconv/run-immune-deconv.sh
+
+          - name: EFO/MONDO annotation
+            entrypoint: efo-mondo-mapping/run_qc.sh
+
+          - name: ENSEMBL Gene matching
+            entrypoint: gene_match/run-gene-mapping.sh
+
+          - name: Update table annotation data
+            entrypoint: long-format-table-utils/run-update-long-format-table-utils.sh
+
+          - name: Fusion frequency tables
+            entrypoint: fusion-frequencies/run-frequencies.sh
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Cache Data
+        uses: actions/cache@v3
+        with:
+          path: ./data
+          key: ${{ runner.os }}-data-${{ hashFiles('download-data.sh') }}
+          restore-keys: |
+            ${{ runner.os }}-data-
+
+      - name: Download Data
+        uses: docker://pgc-images.sbgenomics.com/d3b-bixu/open-pedcan:analysisjob
```

Comment:
If another workflow runs and overwrites the  `analysisjob` tag, some jobs may pull a different version of the build container. To fix this, we should tag the image with the current SHA (e.g., `${{ github.sha }}`). However, I ran into issues where I couldn't get GitHub Actions to accept string interpolation in the `uses` field. Let's return to this problem in future iterations of the workflow.

Reply:
Yea, I ran into similar issues and think it's related to the [github actions contexts](https://docs.github.com/en/actions/learn-github-actions/contexts), and which ones are made available where, and that part is not entirely clear from the docs. :(